### PR TITLE
Package gremlin.0.1.0

### PR DESCRIPTION
**Pull request by bramford**
### `gremlin.0.1.0`
Gremlin Client Library
This is an Apache Tinkerpop3 Gremlin client library.

See the official tinkerpop3 and gremlin docs:
- http://tinkerpop.apache.org/docs/current/

This client library is implemented following the driver provider requirements:
- http://tinkerpop.apache.org/docs/current/dev/provider/#_graph_driver_provider_requirements

---
* Homepage: https://github.com/bramford/ocaml-gremlin
* Source repo: git+https://github.com/bramford/ocaml-gremlin
* Bug tracker: https://github.com/bramford/ocaml-gremlin/issues

---
:camel: Pull-request generated by opam-publish v2.0.0

### CODE REVIEW

File: packages/gremlin/gremlin.0.1.0/opam [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+maintainer: "Bramford Horton <bram.horton@gmail.com>"
+homepage: "https://github.com/bramford/ocaml-gremlin"
+bug-reports: "https://github.com/bramford/ocaml-gremlin/issues"
+dev-repo: "git+https://github.com/bramford/ocaml-gremlin"
+doc: "https://bramford.github.io/ocaml-gremlin/doc"
+license: "ISC"
+build: [ "dune" "build" "-j" jobs "-p" name ]
+depends: [
+  "ocaml" {>= "4.07.1"}
+  "dune" {build & >= "1.10.0"}
```
Comment (kit-ty-kate) [inline packages/gremlin/gremlin.0.1.0/opam]:
```suggestion
  "dune" {>= "1.10.0"}
```
Dune should not be flagged as a build dependency anymore (see #14266)

File: packages/gremlin/gremlin.0.1.0/opam [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  "lwt" {>= "4.2.1"}
+  "containers" {>= "2.6"}
+  "lwt_ppx" {>= "1.2.2"}
+  "conduit-lwt-unix" {>= "1.4.0"}
+  "yojson" {>= "1.7.0"}
+  "core" {>= "0.12.3"}
```
Comment (kit-ty-kate) [inline packages/gremlin/gremlin.0.1.0/opam]:
```suggestion
  "core" {>= "v0.12.3"}
```
The v prefix for janestreet packages are relevant

File: packages/gremlin/gremlin.0.1.0/opam [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  "ppx_let" {>= "0.12.0"}
```
Comment (kit-ty-kate) [inline packages/gremlin/gremlin.0.1.0/opam]:
```suggestion
  "ppx_let" {>= "v0.12.0"}
```

**Comment by kit-ty-kate** [comment]
Also there is a missing dependency apparently:
```

#=== ERROR while compiling gremlin.0.1.0 ======================================#
# context              2.0.4 | linux/x86_64 | ocaml-base-compiler.4.08.0 | pinned(https://github.com/bramford/ocaml-gremlin/archive/v0.1.0.tar.gz)
# path                 ~/.opam/4.08/.opam-switch/build/gremlin.0.1.0
# command              ~/.opam/4.08/bin/dune build -j 2 -p gremlin
# exit-code            1
# env-file             ~/.opam/log/gremlin-15-4a7a12.env
# output-file          ~/.opam/log/gremlin-15-4a7a12.out
### output ###
# File "src/dune", line 7, characters 4-13:
# 7 |     websocket
#         ^^^^^^^^^
# Error: Library "websocket" not found.
# Hint: try: dune external-lib-deps --missing -j 2 -p gremlin @@default
```

**Comment by bramford** [comment]
I've fixed the issues and opened #14534

**Pull request closed without merging**